persist-cli-tests/src/test/java/io/ballerina/persist/tools/utils/GeneratedSourcesTestUtils.java

```diff
@@ -79,9 +80,10 @@ public static void assertGeneratedSources(String subDir, Command cmd) {
         }
 
         for (Path actualOutputFile: listFiles(Paths.get(GENERATED_SOURCES_DIRECTORY).resolve(subDir))) {
+            errStream.println(actualOutputFile.toString());
             if (actualOutputFile.toString().contains("persist_db_scripts.sql")
                     && (subDir.equals("tool_test_generate_7") || subDir.equals("tool_test_generate_14") ||
-                    subDir.equals("tool_test_generate_15"))) {
+                    subDir.equals("tool_test_generate_15") || subDir.equals("tool_test_generate_5"))) {
```

Comment:
Ideally, test utils functions should be independent of test cases. Let's improve the code later.

Reply:
Actually the thing here is, when we are testing the code for semantic errors, we are compiling the code. even though we skipped the compilation in generation, sql file is created at the testing phase